cmd/zed/zed.d/zed.rc

```diff
@@ -86,6 +86,14 @@
 #ZED_SPARE_ON_IO_ERRORS=1
 
 ##
+# Turn on/off enclosure LEDs when drives get DEGRADED/FAULTED.  This works for
+# device mapper and multipath devices as well.  Your enclosure must be
+# be supported by the Linux SES driver for this to work.
+#
+#ZED_USE_ENCLOSURE_LEDS=1
```

Comment:
My inclination is to also enable this by default since it's safe and people like it when things just work.